options/locale/locale_en-US.ini

```diff
@@ -111,7 +111,7 @@ no_admin_and_disable_registration = You cannot disable user self-registration wi
 err_empty_admin_password = The administrator password cannot be empty.
 err_empty_admin_email = The administrator email cannot be empty.
 err_admin_name_is_reserved = Administrator Username is invalid, username is reserved
-err_admin_name_pattern_not_allowed = Administrator Username is invalid, username is pattern is not allowed
+err_admin_name_pattern_not_allowed = Administrator Username is invalid, the username pattern is not allowed
```

Comment:
```suggestion
err_admin_name_pattern_not_allowed = Administrator Username is invalid, the username matches a reserved pattern
```

Reply:
What about?
```suggestion

Reply:
it's not that "Administrator" is invalid, but rather the username that was provided as the administrator is invalid.

Reply:
Oh, OK. I didn't have the context. I'd prefer `user name` rather than `username` anyway, or at least use lower case for `username`.

Reply:
I've updated my suggestion

Reply:
So the context is about administrator username, not necessarily `Administrator`?